tidb-cloud/tidb-cloud-auditing.md

```diff
@@ -79,23 +87,23 @@ For more information, see [Creating a bucket](https://docs.aws.amazon.com/Amazon
 
 #### Step 3. Enable audit logging
 
-In the TiDB Cloud console, go back to the **Audit Logging** dialog box where you got the TiDB Cloud account ID and the External ID values, and then take the following steps:
+In the TiDB Cloud console, go back to the **Enable Database Audit Logging** dialog box where you got the TiDB Cloud account ID and the External ID values, and then take the following steps:
 
 1. In the **Bucket URI** field, enter the URI of your S3 bucket where the audit log files are to be written.
```

Comment:
```suggestion
1. In the **Bucket URL** field, enter the URI of your S3 bucket where the audit log files are to be written.
```

<img width="363" alt="image" src="https://github.com/user-attachments/assets/f7de9073-17c7-4e00-b3c1-1ba28d2d1fef">

Reply:
This is a wrong modification of the Staging environment. The original name "URI" is retained in the release. So keep "URI" in the document.